packages/cli/tests/images/build.js

```diff
@@ -66,8 +67,7 @@ exports.prerender.heads.route66 = `
 	<meta name="apple-mobile-web-app-capable" content="yes">
 	<link rel="manifest" href="\\/manifest\\.json">
 	<link rel="shortcut icon" href="\\/favicon\\.ico">
-	<link href=\\"/bundle.\\w{5}.css\\" rel=\\"preload\\" as=\\"style\\">
-	<style>html{padding:0;}<\\/style>
```

Comment:
I think this might be a bug, need to check.

Reply:
Yes this is the bug we need to fix in critters. With V2 it stops adding inline does to other routes than index